pypeit/spectrographs/gtc_osiris.py

```diff
@@ -439,6 +450,101 @@ def bpm(self, filename, det, shape=None, msbias=None):
 
         return bpm_img
 
+    def tweak_standard(self, wave_in, counts_in, counts_ivar_in, gpm_in, meta_table, log10_blaze_function=None, debug=False):
```

Comment:
Hi @rcooke-ast , I was look over this PR again and I noticed that you should add the parameter `trim_std_pixs` in this `tweak_standard()` function. See an example [here](https://github.com/pypeit/PypeIt/blob/b624425cfa28972b72eaa05dad65b1e71272df9a/pypeit/spectrographs/keck_mosfire.py#L757-L808).
Could you please do that before merging?

Reply:
Thank you for the quick response and action @rcooke-ast . I don't if you were about to do it anyway, but I want to mention that also the following lines of code need to be added for the parameter to be actually used.
```
        if trim_std_pixs is not None:
            return super().tweak_standard(wave_in, counts_in, counts_ivar_in, gpm_in, meta_table,
                                          trim_std_pixs=trim_std_pixs, log10_blaze_function=log10_blaze_function)
```

Reply:
Quite right, I hadn't spotted that  👍  Now implemented, thanks!